src/Widgets/DataSearchCountWidget/DataSearchCountWidgetClass.ts

```diff
@@ -0,0 +1,17 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataSearchCountWidget = provideWidgetClass(
+  'DataSearchCountWidget',
```

Comment:
* [ ] FYI naming: Wouldn't this widget also work without a search, to show the overall count of some entity, e.g., on a dashboard?